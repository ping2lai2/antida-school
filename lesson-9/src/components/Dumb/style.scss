.dumb {
  width: 200px;
  height: 200px;
  display: flex;
  background: #321;
  justify-content: center;
  align-items: center;
  font-size: 30px;
}
