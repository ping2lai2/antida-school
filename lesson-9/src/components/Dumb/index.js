import React from 'react';
import PropTypes from 'prop-types';

import './style.scss';

const Dumb = ({ text }) => <div className="dumb">{text}</div>;

Dumb.propTypes = {
  text: PropTypes.string.isRequired
};

export default Dumb;
