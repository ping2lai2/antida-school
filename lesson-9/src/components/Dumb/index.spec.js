import React from 'react';
import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';

import Dumb from '../Dumb';

describe('Dumb test', () => {
  it('should render correctly', () => {
    const output = shallow(<Dumb text="smth" />);
    expect(shallowToJson(output)).toMatchSnapshot();
  });
});
