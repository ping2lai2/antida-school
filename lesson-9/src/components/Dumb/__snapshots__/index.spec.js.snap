// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Dumb test should render correctly 1`] = `
<div
  className="dumb"
>
  smth
</div>
`;
